src/zjs_event.c

```diff
@@ -266,21 +280,22 @@ static jerry_value_t get_event_names(const jerry_value_t function_obj,
                                      const jerry_value_t argv[],
                                      const jerry_length_t argc)
 {
-    event_t *ev;
     event_names_t names;
 
-    jerry_value_t event_emitter = zjs_get_property(this, HIDDEN_PROP("event"));
-    if (!jerry_get_object_native_handle(event_emitter, (uintptr_t*)&ev)) {
-        jerry_release_value(event_emitter);
-        ERR_PRINT("native handle not found\n");
-        return ZJS_UNDEFINED;
+    if (argc) {
```

Comment:
For better or worse, normally we ignore extra arguments I think. Maybe that's okay?